aten/src/ATen/native/cpu/FillKernel.cpp

```diff
@@ -17,7 +17,7 @@ static void fill_non_native_type(TensorIterator& iter, Scalar value_scalar) {
   using H = typename std::make_signed<decltype(value)>::type;  // Signed type has more acceleration
   // Reserve the representation of value. static_cast<H>(value) is implementation defined.
   H val = *reinterpret_cast<H*>(std::addressof(value));
-  cpu_kernel_vec(
+  cpu_kernel_vec</*check_dynamic_cast=*/false>(
```

Comment:
lol. Do you think comment makes sense here?

Reply:
sure, I can add a comment.  There's a reinterpret_cast the line directly above though!